docs/index.html

```diff
@@ -0,0 +1,3525 @@
+<!doctype html>
```

Comment:
Whatever tool we're using is creating some really bad HTML styling (bad indentation, bad nesting, extraneous newlines, etc). Not sure if we should try to fix this in the tooling.